ReplaySample/src/main/res/layout/activity_masking.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ScrollView
+        android:id="@+id/scrollView"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:layout_gravity="center"
+        >
+
+        <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:paddingBottom="100dp"
+            >
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="10dp"
+                android:layout_marginBottom="5dp"
+                android:layout_marginTop="5dp"
+                android:textAlignment="center"
+                android:gravity="fill_horizontal"
+                android:text="@string/warning_recording_to_user" />
```

Comment:
This can be removed since we're not going to have a shared account. Instead, it should have some instructions, e.g.
> Input fields are masked automatically by the SDK, but you have full control of the masking and unmasking of any views. You can use the toggles below to set the masking status for each view. 
> 
> In this app, debug masking is enabled so that you can verify the position of the masks you set using the red overlays. These overlays are not present when debug masking is not enabled.